airflow/providers/cncf/kubernetes/backcompat/pod.py

```diff
@@ -15,9 +15,18 @@
 # specific language governing permissions and limitations
 # under the License.
 """Classes for interacting with Kubernetes API"""
+"""This module is deprecated. Please use :mod:`kubernetes.client.models.V1ResourceRequirements` and `kubernetes.client.models.V1ContainerPort`."""
```

Comment:
```suggestion
"""This module is deprecated. Please use :mod:`kubernetes.client.models.V1ResourceRequirements` and :mod:`kubernetes.client.models.V1ContainerPort`."""
```

(Sorry, I edited this into late in my last suggestion it seems)